services-js/311/components/common/TelephoneNumbers.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { assetUrl, MEDIA_LARGE } from '../style-constants';
+
+const SPRITE_URL = assetUrl('img/svg/faq-icons.svg');
+
+const CONTAINER_STYLE = css({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const TELEPHONE_STYLE = css({
+  display: 'none',
+  width: 36,
+  height: 36,
+  marginRight: '1rem',
+  [MEDIA_LARGE]: {
+    display: 'block',
+  },
+});
+
+/**
+ * Component that adds a little telephone icon and information about how to call
+ * BOS:311 when the website is having issues.
+ */
+export default function TelephoneNumbers(): JSX.Element {
+  return (
+    <div className={CONTAINER_STYLE}>
+      <svg role="img" className={TELEPHONE_STYLE}>
```

Comment:
It’s probably cleaner to swap the `role` attribute for `aria-hidden="true"`?

Reply:
Done. I'm curious to learn what makes `aria-hidden` seem cleaner.

Reply:
When icons are used solely as decoration, my opinion is that it's better to exclude them from screen readers entirely. Assigning a role is sort of the opposite effect.

Reply:
👍